conn2res/reservoir.py

```diff
@@ -348,7 +348,7 @@ def solveVi(self, Ve, Vgr=None, G=None, **kwargs):
         # inverse matrix A_II
         A_II = A[np.ix_(self._I, self._I)]
         # print(matrix_rank(A_II, hermitian=check_symmetric(A_II)))
-        A_II_inv = inv(A_II)
+        A_II_inv = pinv(A_II)
```

Comment:
I was getting errors from trying to invert a singular matrix (maybe multiple rows of all zeroes?). So I switched to the pseudoinverse to avoid this problem.